.circleci/config.yml

```diff
@@ -1708,7 +1726,36 @@ jobs:
       - store_artifacts:
           path: /results.tar.gz
 
-  weekly-run-start-from-saved-state-regression:
+  weekly-run-crypto-transfer-start-from-saved-state-regression:
+    parameters:
+      workflow-name:
+        type: string
+        default: ""
+    executor:
+      name: build-executor
+      workflow-name: << parameters.workflow-name >>
+    steps:
+      - run:
+          name: Modify Payer account for Public testnet start from saved state test
+          command: |
+            echo -n "$KEY_DevTestNetTreasury" > /repo/test-clients/src/main/resource/StartUpAccount.txt;
+      - run:
+          name: Run start from saved state regression tests
+          no_output_timeout: 60m
+          command: |
+            cd /swirlds-platform/regression;
+            ./regression_services_circleci.sh configs/services/weekly/15N_15C/AWS-Services-Weekly-CryptoTransfer-Perf-StartFromState-15N-15C.json /repo
```

Comment:
`AWS-Services-Weekly-CryptoTransfer-Perf-StartFromState-15N-15C.json` naming convention should be changed as mentioned in other PR

Reply:
I agree. But since the Circleci CI workflow still not stable and I don't want to take risk to make the changes and then couldn't pass the CI check before the weekend. I will convert them to be weekly regression. At that time, I think I will change all the names following the new name convention. Hopefully over the weekend, these tests should be running fine. So for now I will stick to the old names. 
Please also refer to the comment I have for the other PR.